pages/sitecontent.py

```diff
@@ -10,8 +10,8 @@
 
 class SiteContentPermissionPolicy(ModelPermissionPolicy):
     def user_has_permission(self, user, action):
-        # Disable creating new site content instances
-        if action == 'change':
+        # Disable creating and deleting of site content instances
+        if action in ('add', 'delete'):
```

Comment:
Is preventing delete the right thing to do? This seemed like the kind of thing that should never be deleted and that there should always be one and only one instance available.